detox/e2e/support/server_api/bot.ts

```diff
@@ -26,7 +26,7 @@ import {getResponseFromError} from './common';
  * @param {Object} option.bot - bot object to be created
  * @return {Object} returns {bot} on success or {error, status} on error
  */
-export const apiCreateBot = async (baseUrl, {prefix = 'bot', bot = null} = {}) => {
+export const apiCreateBot = async (baseUrl: string, {prefix = 'bot', bot = null}: any = {}): Promise<any> => {
```

Comment:
Can `any` be avoided?

Reply:
I can probably revisit this later on a future PR as this might need more thought on whether to have types that would only be need within the context of api parameters for different api calls.